### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by Tomislav3383** [Wurst-Imperium/Wurst-Client#1969]
I was trying to join a server and this happened, after a client restart the problem disappeared.

```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// I feel sad now :(

Time: 23.08.15. 19:41
Description: Ticking screen

java.lang.NullPointerException: Ticking screen
	at net.minecraft.network.NetworkManager$InboundHandlerTuplePacketListener.access$0(NetworkManager.java:454)
	at net.minecraft.network.NetworkManager.flushOutboundQueue(NetworkManager.java:251)
	at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:260)
	at net.minecraft.client.multiplayer.GuiConnecting.updateScreen(GuiConnecting.java:115)
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1964)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.network.NetworkManager$InboundHandlerTuplePacketListener.access$0(NetworkManager.java:454)
	at net.minecraft.network.NetworkManager.flushOutboundQueue(NetworkManager.java:251)
	at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:260)
	at net.minecraft.client.multiplayer.GuiConnecting.updateScreen(GuiConnecting.java:115)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.multiplayer.GuiConnecting
Stacktrace:
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1964)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
	at net.minecraft.client.Minecraft.run(Minecraft.java:383)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 2.3 (latest: 2.3)
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 241125208 bytes (229 MB) / 411549696 bytes (392 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 960/PCIe/SSE2 GL version 4.5.0 NVIDIA 355.60, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
```

**Comment by Elephantlovr33X** [comment]
Have you tried re installing wurst?

**Comment by WurstContributor** [comment]
Potential Duplicate of #1803. It's A rare crash.

**Comment by JackFishbaugh** [comment]
```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// I feel sad now :(

Time: 23.08.15. 19:41
Description: Ticking screen

java.lang.NullPointerException: Ticking screen
at net.minecraft.network.NetworkManager$InboundHandlerTuplePacketListener.access$0(NetworkManager.java:454)
at net.minecraft.network.NetworkManager.flushOutboundQueue(NetworkManager.java:251)
at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:260)
at net.minecraft.client.multiplayer.GuiConnecting.updateScreen(GuiConnecting.java:115)
at net.minecraft.client.Minecraft.runTick(Minecraft.java:1964)
at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
at net.minecraft.client.Minecraft.run(Minecraft.java:383)
at net.minecraft.client.main.Main.main(Main.java:127)

A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
at net.minecraft.network.NetworkManager$InboundHandlerTuplePacketListener.access$0(NetworkManager.java:454)
at net.minecraft.network.NetworkManager.flushOutboundQueue(NetworkManager.java:251)
at net.minecraft.network.NetworkManager.processReceivedPackets(NetworkManager.java:260)
at net.minecraft.client.multiplayer.GuiConnecting.updateScreen(GuiConnecting.java:115)

-- Affected screen --
Details:
Screen name: net.minecraft.client.multiplayer.GuiConnecting
Stacktrace:
at net.minecraft.client.Minecraft.runTick(Minecraft.java:1964)
at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1272)
at net.minecraft.client.Minecraft.run(Minecraft.java:383)
at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
Minecraft Version: 1.8
Wurst Client Version: 2.3 (latest: 2.3)
Operating System: Windows 10 (amd64) version 10.0
Java Version: 1.8.0_25, Oracle Corporation
Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 241125208 bytes (229 MB) / 411549696 bytes (392 MB) up to 1060372480 bytes (1011 MB)
JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
Launched Version: Wurst
LWJGL: 2.9.1
OpenGL: GeForce GTX 960/PCIe/SSE2 GL version 4.5.0 NVIDIA 355.60, NVIDIA Corporation
GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

Using VBOs: No
Is Modded: Very likely; Jar signature invalidated
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
```

**Comment by MyzticalTekkno** [comment]
<b><i>Duplicate #1803</b></i>
